repositories.bzl

```diff
@@ -8,9 +11,9 @@ def load_com_google_googletest():
     )
 
 def load_com_github_nelhage_rules_boost():
-    native.git_repository(
+    git_repository(
         name = "com_github_nelhage_rules_boost",
-        commit = "8cbc2be26be786d6f850fcf229dc06d158a79d81",
+        commit = "6d6fd834281cb8f8e758dd9ad76df86304bf1869",
```

Comment:
We needed to update this dependency since it needed to make changes to it's own Bazel BUILD files to update calls to `http_archive` as well.